exonum-java-binding-core/rust/src/runtime/config.rs

```diff
@@ -22,23 +22,19 @@ pub struct JvmConfig {
     pub system_class_path: String,
     /// Java service classpath.
     pub service_class_path: String,
-    /// Path to java-bindings shared library.
-    ///
-    /// Should be path to exonum-java-binding-core/rust/target/{debug, release}
-    pub lib_path: String,
```

Comment:
Also integration tests seems not adopted for removed `lib_path` and `ServiceConfig.port`

Reply:
Oh yep, indeed :upside_down_face: